### DISCUSSION TITLE
how to load the data into indexedDb to load a new page into canvas?

### DISCUSSION THREAD

**Discussion by senthil177cse** [GrapesJS/grapesjs#4395] [Q&A]
Hi @artf ....
    i am using indexeddb to save data using grapesjs. i have data for different page so that data i need to load into gjs indexeddb and show into the canvas. can you pls tell me how to do that one?

**Comment by senthil177cse** [comment]
Hi @artf  Any update of this please?

**Reply by artf** [reply]
If you're not using built-in pages (where multiple pages are already stored in the JSON) probably the best approach would be to redirect your editor from one project to another (eg. `/editor-page/PROJECT-ID` > `/editor-page/PROJECT-ID-2`). I know the explanation is quite high-level but this is something strictly related to your app implementation.
